### DISCUSSION TITLE
Rain today amounts different in HA than weatherflow app

### DISCUSSION THREAD

**Discussion by beyondunreal79** [briis/hass-weatherflow2mqtt#247] [Q&A]
I am using the HA addon. The rain today amount is showing as much higher in HA than the actual app. See pics. Where is that higher value coming from.
![image](https://github.com/briis/hass-weatherflow2mqtt/assets/45919008/ce8ef775-9e70-4e3e-847d-9587172d0c72)

![image](https://github.com/briis/hass-weatherflow2mqtt/assets/45919008/3a75469c-8ac4-4fbf-8ef8-d8fcd5250522)

**Comment by briis** [comment]
This Add-On can only pull the raw data from the station as it is only local. There is no 'Daily Rain Total' value available as local data, so the Add-On starts summarizing the data recorded when the Add-On starts and then resets by midnight.
If you have Nearcast enabled that will mostly give you a different value, as it corrects the rain data based on nearby stations where this Add-On cannot do that as it is pure local.

**Reply by beyondunreal79** [reply]
If nearcast is disabled, would it be the same as the app then? Is there a template sensor or some way to convert that rain valve into an actual amount?  I also noticed something else.  A few times now in HA, it shows lightning strikes, but in the weatherflow app, none is shown. Where is that info coming from? 
![image](https://github.com/briis/hass-weatherflow2mqtt/assets/45919008/c60af332-78c8-420a-8ee9-7f335303e4d5)

![image](https://github.com/briis/hass-weatherflow2mqtt/assets/45919008/86b99489-c341-43c4-9292-f240b4ca82f7)

**Reply by briis** [reply]
I can't really see a possibility to create a template sensor to correct that, without pulling data from other weather stations in your area, and then trying to do what Nearcast does - and that task is not possible here. If you want that data, you should [install my other WeatherFlow integration](https://github.com/briis/weatherflow_forecast) that pulls the data, using the REST API - that integration shows the same values as the APP.

Lightning: As with the rain values, this integration reports the raw data as they come in - I would assume that WeatherFlow servers have some kind of algorithm that can filter out wrong readings, but I am in no position to apply this here.
